release-notes.txt

```diff
@@ -0,0 +1,237 @@
+BISDN Linux
+v4.5.0 Release Notes
+Date: 2022-03-15
+
+! New in this release
+- baseboxd:
+  - updated version to 1.11.1
+  - added support for Spanning Tree Protocol (STP, RSTP and MSTP) on bonded interfaces
+  - added initial support for Broadcom KNET interfaces
+- Linux kernel updated to version 5.15.22
+- iproute2: udpated to version 5.16
```

Comment:
might be worthwhile to add a one liner as ! Changed behaviour that you now need to use `bridge -d vlan` instead of `bridge vlan database` to see per-vlan stp states.

Reply:
Added section on changed behaviour